src/modules/teams/components/RegisterUserList.vue

```diff
@@ -205,6 +206,7 @@ handleAllUsers();
           <th scope="row" v-text="index +1" />
           <td v-text="member.name" />
           <td v-text="member.email" />
+          <td>{{member.role[0].role.toUpperCase()}}</td>
```

Comment:
Use computed value to convert to uppercase, instead of doing it on the template.